apps/editing-toolkit/package.json

```diff
@@ -40,10 +40,8 @@
 		"lint:php": "../../vendor/bin/phpcs --standard=../phpcs.xml ./ ",
 		"lint:php:fix": "../../vendor/bin/phpcbf --standard=../phpcs.xml ./ ",
 		"sync:newspack-blocks": "./bin/sync-newspack-blocks.sh",
-		"test:js": "npx wp-scripts test-unit-js --config='jest.config.js' --colors",
-		"test:js:help": "npx wp-scripts test-unit-js --config='jest.config.js' --help --colors",
-		"test:js:update-snapshots": "npx wp-scripts test-unit-js -u --config='jest.config.js' --colors",
-		"test:js:watch": "npx wp-scripts test-unit-js --config='jest.config.js' --watch --colors",
+		"test:js": "yarn run -T test-apps apps/editing-toolkit",
```

Comment:
`-T` executes the script from the workspace root. In this case, it's necessary because `babel-jest` won't work unless it can find a babel config file. (And without that, imports don't work!) That config file is found in the root of the repo. There doesn't seem to be an easy way to configure this, other than running jest from the same directory. Running unit tests from the root package.json seems to solve that.